openid-connect-key-binding-1_0.md

```diff
@@ -215,26 +215,43 @@ Non-normative example of the ID Token payload:
 }
 ```
 
-# Privacy Considerations
+## ID Token Proof of Possession
+
+The mechanism for how an RP authenticating component proves to an RP consuming component that it possesses the private keys associated with the `cnf` claim in the ID Token is out of scope of this document.
+
+> If the WG wants to, we can also profile how to use KB to bind a proof of possession to an ID Token for presentation when a proof of possesion is not present.
 
-Public key bound ID Tokens will often contain personal (PII). The RP SHOULD obtain user consent before sharing a Public key bound ID Token that contains PII with a third party.
+# Privacy Considerations
 
-*To be completed.*
+An RP authenticating component SHOULD only share an ID Token with a consuming component when such sharing is consistent with the original purpose for which the PII was collected and the scope of consent obtained from the user.
 
 # Security Considerations
 
-Proof of possession authentication provides a greater level of security than bearer token authentication. To authenticate with a bearer token, the authentication secret must be sent over the internet to the authenticating party. This presents a risk that the authentication secret be stolen is transit or stolen at the server endpoint and replayed. With proof of possession, the authentication secret, i.e. the private key, never needs to leave the client. This reduces the chance of exposure and allows the client to use additional security mechanisms to protect the private key such as HSMs (Hardware Security Modules) or web browser based SSMs (Software Security Modules).
```

Comment:
Did you intend to cut this section? I do not object if you did.

Reply:
yes -- I did intend to cut -- it does not read as a consideration, but as a statement of the benefits